proxy/src/auth/backend/jwt.rs

```diff
@@ -324,6 +383,21 @@ struct JWTHeader<'a> {
     kid: Option<&'a str>,
 }
 
+/// <https://datatracker.ietf.org/doc/html/rfc7519#section-4.1>
+#[derive(serde::Deserialize, serde::Serialize, Debug)]
+struct JwtPayload<'a> {
+    /// Audience - Recipient for which the JWT is intended
+    aud: Option<&'a str>,
+    /// Expiration - Time after which the JWT expires
```

Comment:
Should mention seconds (since epoch) as unit. I wonder if it's better to keep this struct pure like this or let serde help make it more convenient with `#[serde(rename=)]` and type for timestamps.

Reply:
I think if we go that route it would b better to use `#[serde(deserialize_with)]` to turn the u64 directly into SystemTime types

Reply:
Yes, exactly. Or some special PayloadTimestamp(u64) type that can be converted into SystemTime if/when needed.

Reply:
Switched to a `numeric_date_opt` deser function, to match the `NumericDate` type name in the JWT RFC. It returns a SystemTime for convenience